server/config/config.go

```diff
@@ -151,11 +151,17 @@ type RemoteExecutionConfig struct {
 }
```

Comment:
can you revert this part of this CL? (see my comment in https://github.com/buildbuddy-io/buildbuddy-internal/pull/206)